content/applications/sales/subscriptions/reports.rst

```diff
@@ -36,26 +36,28 @@ organized analysis.
 This section refers to both filters and groupings, as a combination of the two can be saved in the
 :guilabel:`Favorites` section.
 
-To modify the results being shown on any reporting page, click the :guilabel:`▼ (downward facing
-arrow)` icon to the right of the search bar. Doing so reveals a drop-down menu of detailed filter
-and grouping options.
+To modify the results being shown on any reporting page, click the :icon:`fa-caret-down`
+:guilabel:`(down arrow)` icon to the right of the search bar. Doing so reveals a drop-down menu of
+detailed filter and grouping options.
 
 .. note::
    Multiple filter options and grouping options can be selected at once.
 
-.. image:: reports/reporting-filter-options-dropdown.png
+.. figure:: reports/reporting-filter-options-dropdown.png
    :align: center
    :alt: The reporting filter drop-down menu of options in the Odoo Subscriptions application.
 
+   The reporting filter drop-down menu of options in the Odoo Subscriptions application.
```

Comment:
It might not be strictly necessary, but it was included when I was shown how to use figures by Felicia. I believe it might still be beneficial to people using assisted reading devices to have the alt text compared to only captions.